core/src/smartcontracts/isi/triggers/set.rs

```diff
@@ -346,22 +353,37 @@ impl Set {
         let loaded_executable = match executable {
             Executable::Wasm(bytes) => {
                 let hash = HashOf::new(&bytes);
-                let loaded = LoadedExecutable::Wasm(LoadedWasm {
-                    module: wasm::load_module(engine, &bytes)?,
-                    blob_hash: hash,
-                });
                 // Store original executable representation to respond to queries with.
-                self.original_contracts
-                    .entry(hash)
-                    .and_modify(|(_, count)| {
+                let module = match self.original_contracts.entry(hash) {
```

Comment:
I think this should no longer be `original_contracts`, but just `contracts` or maybe `wasm_executables`?

Reply:
Decided to use `contracts`